src/transformers/commands/pt_to_tf.py

```diff
@@ -196,7 +196,7 @@ def __init__(
         self._extra_commit_description = extra_commit_description
         self._override_model_class = override_model_class
 
-    def get_inputs(self, pt_model, config):
+    def get_inputs(self, pt_model, tf_dummy_inputs, config):
```

Comment:
The changes here seem unrelated to this PR and would be better in their own PR, no?

Reply:
Fair! I added them because they were needed for the `pt-to-tf` code to port the BLIP models correctly. If you'd rather I move them to a separate PR though, that's fine!